backend/src/api/tripadvisorAPI.service.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import path from 'path';
+import axios from 'axios';
+
+const reviewsFilePath = path.join(__dirname, 'reviews.json');
+const options = {
+    method: 'GET',
+    url: 'https://tripadvisor16.p.rapidapi.com/api/v1/restaurant/searchRestaurants',
+        params: {
+        locationId: '293984'
+      },
+      headers: {
+        'X-RapidAPI-Key': 'f8662f8ed1msh343418130ebdb98p11bf57jsna9bf16098028',
+        'X-RapidAPI-Host': 'tripadvisor16.p.rapidapi.com'
+      }
+};
+
+const fetchReviews = async () => {
+    try{
+        const response = await axios.request(options);
+        fs.writeFileSync(reviewsFilePath, JSON.stringify(response.data.data.data, null, 2));
+        console.log('File written successfully');
+    } catch (error) {
+        console.error(error);
+    }
+}
+
+export default { fetchReviews };
```

Comment:
Why isnt this in the services folder? You can move that api folder there.

Also what about the model for restaurants and posts? The data fetched from the API should be manipulated to fit the model and for easy use.

Reply:
the next branch with the next PR will have this functionality.
i need to figure out how to manipulate that data into the DB, with the corresponding models.
also, develop that with TDD.